apps/pay/app/globals.css

```diff
@@ -1,17 +1,13 @@
-@import url("https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@200;300;400;600;700&display=swap");
```

Comment:
I don't think it's necessary to have the `_` in `_reducer.tsx` same for `_user.module.css` anymore. I would guess this is an artefact of the way the `page` router worked before

(commenting here because I can't comment on a file that has only a file name change)

Reply:
yup, I am going to clean these.